resources/banter/index.html

```diff
@@ -62,76 +62,61 @@
         <div id="intro" class="section-panel">
           <div class="row">
             <div class="col-sm-12">
-              <h2>Introduction</h2>
-              <h3>Banter is…</h3>
-              <p>an Olin alumni-run program designed to connect current students and alumni to share experience. Newly expanded to connect alumni with alumni !</p>
-              <h4>Banter</h4>
-              <p>noun.</p>
-              <ul>
-                <li>a chance to build a lasting relationship with a member of the Olin alumni community, while receiving advice such as:
+              <h2>Banter</h2>
+              <h4>TL;DR.</h4><p>Meet an <strong>Olin alumnus</strong>. Find an <strong>awesome mentor</strong>. Discuss life at Olin, how to adult, what you did last week, and everything in between. Hindsight is 20/20! Borrow some perspective from someone who's been there.</p>
```

Comment:
Can you put each block level tag on its own line? In this case, linebreak after the `</h4>`.